tests/integration/botocore/test_smoke.py

```diff
@@ -273,11 +271,8 @@ def test_can_make_request_with_client(
     # instead of service/operations.
     client = _get_client(botocore_session, service_name)
     method = getattr(client, xform_name(operation_name))
-    with warnings.catch_warnings(record=True) as caught_warnings:
-        response = method(**kwargs)
-        err_msg = f"Warnings were emitted during smoke test: {caught_warnings}"
-        assert len(caught_warnings) == 0, err_msg
-        assert 'Errors' not in response
```

Comment:
Why do we need to do this? The PR description doesn't go into what this is fixing. It just looks like we're getting less coverage. It looks like [botocore never removed the warnings assertion here](https://github.com/boto/botocore/blob/develop/tests/integration/test_smoke.py#L286). We should avoid drifting if it's not necessary.